src/utils/roll-build-images.ts

```diff
@@ -0,0 +1,164 @@
+import * as debug from 'debug';
+
+import { MAIN_BRANCH, REPOS } from '../constants';
+import { getOctokit } from './octokit';
+import { PullsListResponseItem } from '../types';
+import { Octokit } from '@octokit/rest';
+
+export async function getFileContentFromBuildImages(
+  octokit: Octokit,
+  filePath: string,
+  ref = MAIN_BRANCH,
+) {
+  const { data } = await octokit.repos.getContent({
+    ...REPOS.buildImages,
+    path: filePath,
+    ref,
+  });
+  if ('content' in data) {
+    return { raw: Buffer.from(data.content, 'base64').toString('utf8'), sha: data.sha };
+  }
+  throw new Error(`Failed to get content for ${filePath}`);
+}
+
+function getBuildImagesPRText(bumpSubject: string, newShortVersion: string, diffLink: string) {
+  return {
+    title: `build: bump ${bumpSubject} to ${newShortVersion.substring(0, 12)}`,
+    body: `Updating ${bumpSubject} to \`${newShortVersion}\`
+
+See [changes in Chromium](${diffLink})`,
+  };
+}
+
+export async function rollBuildImages(
+  rollKey: string,
+  bumpSubject: string,
+  previousSha: string,
+  newSha: string,
+  filePath: string,
+  newContent: string,
+): Promise<any> {
+  const d = debug(`roller/build-images/${rollKey}:rollBuildImages()`);
+  const octokit = await getOctokit();
+
+  const branchName = `roller/build-images/${rollKey}`;
+  const shortRef = `heads/${branchName}`;
+  const ref = `refs/${shortRef}`;
+
+  const { owner, repo } = REPOS.buildImages;
+
+  const diffLink =
+    `https://chromium.googlesource.com/chromium/src/+log/` +
```

Comment:
```suggestion
    'https://chromium.googlesource.com/chromium/src/+log/' +
```

suggestion(non-blocking): this could be single quotes rather than backticks since there's no substitution in this part which is just being concatenated.